packages/remix-dev/cli/create.ts

```diff
@@ -188,7 +188,8 @@ async function extractLocalTarball(
     throw Error(
       "🚨 There was a problem extracting the file from the provided template.\n\n" +
         `  Template filepath: \`${filePath}\`\n` +
-        `  Destination directory: \`${projectDir}\``
+        `  Destination directory: \`${projectDir}\`\n` +
+        `  ${err}`
```

Comment:
This test is passing locally and failing on this error. Happy to remove this after I find out why it's failing on GH actions, but I think the error itself should be somewhere in this error message